### PR TITLE
Added basic/initiall OC support

### PR DESCRIPTION
**Pull request by dbarrosop**
DO NOT MERGE!!! WIP

Depends on `oc-poc` branches of `napalm-eos` and `napalm-yang`.

The idea of this is to start playing with the OC functionality of napalm. So far it implements a way of mapping text to OC so the `config` sections of the OC model can be implemented. Example in napalm-automation/napalm-eos#123

### CODE REVIEW

File: napalm_base/base.py [hunk 1]
```diff
@@ -1499,3 +1506,55 @@ def compliance_report(self, validation_file='validate.yml'):
...
         """
         return validate.compliance_report(self, validation_file=validation_file)
+
+    def _oc_all_config(self):
+        raise NotImplementedError
+
+    def oc_populate_interfaces(self):
+        self.interfaces = napalm_yang.oc_if.Interfaces().interfaces
+
+        with open("napalm_eos/openconfig_mappings.yaml", "r") as f:
```
Comment (mirceaulinic) [inline napalm_base/base.py]: Is this intentional?

Comment (dbarrosop) [inline napalm_base/base.py]: yeah, sort of, mostly because I am testing and I didn't want to bother fixing the dynamic path. It has to be fixed for sure.

Comment (dbarrosop) [inline napalm_base/base.py]: It will also require fixes on the MANIFEST file

Comment (mirceaulinic) [inline napalm_base/base.py]: Nice!

**Pull request closed without merging**